### PR TITLE
LPD-31382

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
resend of https://github.com/liferay-commerce/liferay-portal/pull/5143

---

https://liferay.atlassian.net/browse/LPD-31382

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/quantity_selector/InputQuantitySelector.js [hunk 1]
```diff
@@ -84,9 +89,7 @@ const InputQuantitySelector = forwardRef(
 				quantity,
 				unitOfMeasure ? min : Math.ceil(min),
 				unitOfMeasure ? max : Math.ceil(max),
-				unitOfMeasure
-					? unitOfMeasure.incrementalOrderQuantity
-					: Math.ceil(step)
+				unitOfMeasure ? step : Math.ceil(step)
```
Comment (smottal) [inline modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/quantity_selector/InputQuantitySelector.js:92]: why in this case we are using only the step and not the getMultiple(....) ?
I mean... we replaced the `unitOfMeasure.incrementalOrderQuantity` with `getMultiple(....` in the other places but here we completely removed it to use step

Comment (andrea-ale-sbarra) [reply modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/quantity_selector/InputQuantitySelector.js:92]: not sure I've just refactored the code I saw from the other PR, will look at this

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Pull request closed without merging**